projects/driver_controls/src/main.c

```diff
@@ -1,131 +1,87 @@
-#include <stdbool.h>
+
```

Comment:
Please revert this main. Use a test to work in. Don't overwrite the main program.